content/en/get-started/store-and-retrieve/troubleshooting/index.md

```diff
@@ -23,19 +23,20 @@ Reset your Lotus full-node computer if you are still facing issues after editing
 
 Run `echo $SHELL` to find out which shell you are using:
 
-```shell with-output
-echo $SHELL
-```
-```                                                                                                ~
-/bin/zsh
-```
+    ```shell
```

Comment:
We should not add an extra tab/space here. Because tab/space is already recognized as code-block for MD which will make it looks like:
<img width="342" alt="image" src="https://user-images.githubusercontent.com/34025001/188434261-90f3fe34-7441-4b7a-8adb-0e69298d29f2.png">